templates/comment/_macros.html.twig

```diff
@@ -62,18 +64,29 @@
           {% if comment.isStickied %}stickied comment{% endif %}
         </h1>
 
-        <div class="comment-body">
-          {{ comment.body|cached_markdown(markdown_context())|raw }}
-        </div>
+        {% if not comment.softDeleted %}
+          <div class="comment-body">
+            {{ comment.body|cached_markdown(markdown_context())|raw }}
+          </div>
+        {% endif %}
+        {% if comment.softDeleted %}
+          <div class="comment-body">
+            [deleted]
+          </div>
+        {% endif %}
 
         <nav class="comment-nav">
-          <ul>{{ block('comment_nav') }}</ul>
+          <ul class="comment-nav-ul">{{ block('comment_nav') }}</ul>
         </nav>
 
         {% if options.show_form ?? false and is_granted('ROLE_USER') %}
           {% from _self import comment_form %}
           {{ comment_form(comment.submission, comment) }}
         {% endif %}
+
+        {% if is_granted('moderator', comment.submission.forum) and comment.reportCount > 0 %}
```

Comment:
this should've increased number of DB calls, if comments isnt' EAGER to load it's submission and Submission isn't EAGER to load forum - make sure it's so.
Also - let's add cache (see my just landed PR) to comment repo queries if not already.

In test plan - add something like:
1. Prepare Submission with 12 comments from 4 different users
2. Use Profiler to see number of DB calls and performance time in Doctrine
3. Compare before PR and after - it should NOT be significantly higher after

Reply:
I'll check this out before i merge it in and see if the DB calls are any different.

Reply:
Oh, i already merged it lol. I'll check it compared to previous commit.

Reply:
Submission with 12 comments (2 being chained onto another comment)

**Before:**
- 43 queries, not logged in as admin
- 44 queries, logged in ad admin

**After:**
- 43 queries, not logged in as admin (with and without reports)
- 44 queries, logged in ad admin (with and without reports)

It looks like this commit doesn't affect DB calls. I noticed that regardless of this feature, the DB calls went up with each submitted comment. I'm not sure if that is expected, or something that needs to be looked at.

Reply:
Yeah I need to look into that - create an issue and assign to me please?